pkg/collector/iptables_collector_test.go

```diff
@@ -0,0 +1,38 @@
+package collector
+
+import (
+	"testing"
+)
+
+func TestNewIPTablesCollector(t *testing.T) {
+	tests := []struct {
+		name          string
+		want          int
+		wantErr       bool
+		collectorName string
+	}{
+		{
+			name:          "get iptables logs",
+			want:          1,
+			wantErr:       true,
+			collectorName: "iptables",
+		},
+	}
+
+	c := NewIPTablesCollector()
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			err := c.Collect()
+			if (err != nil) == tt.wantErr {
```

Comment:
nit/suggestion: though we already had many tests, will it be better to use testify? https://github.com/stretchr/testify We could save many boilerplate code with this package

Reply:
❤️❤️☕️ Thank you for this, 100 percent yes, yeah I think I will do another dedicated PR for that as a bulk update as one dedicated PR, I am also wishing/hoping of SMI and OSM moves to client-go use then we can pretty much delete the kubectl binary with image as well.